packages/wallet-adapter-react/src/useWallet.tsx

```diff
@@ -24,6 +24,11 @@ export {
   isMobile,
 };
 
+export interface TransactionOptions {
```

Comment:
I think this interface can be placed in the `core` package [types file](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/types.ts) then we can export the type and use it in the react package.

Reply:
commit [8230ee7](https://github.com/aptos-labs/aptos-wallet-adapter/pull/156/commits/8230ee71c9ad6a41e67bcd6a4d5c451125ab4d45)